### PR TITLE
Check for duplicates

### PR DESCRIPTION
**Pull request by Robstaa**

### CODE REVIEW

File: app/jobs/link_check_job.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      check_link(funding)
+      sleep 1
+    end
+  end
+
+  private
+
+  def check_link(funding)
+    response = http_request(funding.link)
+    return if response.is_a? Net::HTTPSuccess
```
Comment (djmaze) [inline app/jobs/link_check_job.rb:18]: Mhh, could be 302, 307 or 308 as well, which would not really indicate a broken link.

File: app/views/businesses/check_duplicates.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<%- model_class = Business -%>
+<div class="page-header">
+  <h1><%=t '.title', :default => model_class.model_name.human.pluralize.titleize %></h1>
+  <% if @business_import -%>
```
Comment (djmaze) [inline app/views/businesses/check_duplicates.html.erb:4]: This is not set in the controller action. Probably makes no sense either here.

File: app/views/businesses/check_duplicates.html.erb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      <th><%= model_class.human_attribute_name(:name) %></th>
+      <th>Ids</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @businesses.each do |business| %>
+      <tr>
+        <td><%= business.name %></td>
+        <td>
+          <% Business.where(name: business.name).each do |duplicate_business| %>
```
Comment (djmaze) [inline app/views/businesses/check_duplicates.html.erb:22]: Should probably filter on `city` as well?

File: app/jobs/link_check_job.rb (hunk 1 above)
Comment (Robstaa) [reply app/jobs/link_check_job.rb:18]: Resolved in f8f1485

File: app/views/businesses/check_duplicates.html.erb (hunk 2 above)
Comment (Robstaa) [reply app/views/businesses/check_duplicates.html.erb]: Resolved in a75fde9

File: app/views/businesses/check_duplicates.html.erb (hunk 3 above)
Comment (Robstaa) [reply app/views/businesses/check_duplicates.html.erb]: Resolved in 8414c4c

**Pull request merged**